app/src/main/res/layout/item_creating_left_conversation.xml

```diff
@@ -53,18 +56,25 @@
             app:layout_constraintStart_toEndOf="@id/image_recording_audio"
             bind:onTouchListen="@{viewModel.getTouchListener(position)}" />
 
-        <ImageView
+        <TextView
             android:id="@+id/imageView4"
-            android:layout_width="@dimen/dp_24"
-            android:layout_height="@dimen/dp_24"
-            android:layout_marginEnd="@dimen/dp_8"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/dp_4"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 16) you should *also* add `android:layout_marginRight="@dimen/dp_4"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.